src/cli.rs

```diff
@@ -182,6 +182,14 @@ Example: --print=Hb"
     #[clap(short = 'S', long = "stream", name = "stream")]
     pub stream_raw: bool,
 
+    ///  Content compressed (encoded) with Deflate algorithm.
+    ///  The Content-Encoding header is set to deflate.
+    ///
+    ///  Compression is skipped if it appears that compression ratio is negative.
+    ///  Compression can be forced by repeating this option.
```

Comment:
Should we also mention that compression cannot be forced if the `Content-Encoding` request header is present?